austrakka/components/iam/scope/__init__.py

```diff
@@ -0,0 +1,21 @@
+import click
```

Comment:
its saying this is unused is that a bug rider is doing something funny perhaps.

Reply:
Strange. Will check but it is being used.